qt_gui_cpp/src/qt_gui_cpp_sip/CMakeLists.txt

```diff
@@ -31,6 +31,39 @@ set(qt_gui_cpp_sip_INCLUDE_DIRS ${qt_gui_cpp_INCLUDE_DIRS} "${CMAKE_CURRENT_SOUR
 set(qt_gui_cpp_sip_LIBRARY_DIRS ${qt_gui_cpp_LIBRARY_DIRS} lib)
 set(qt_gui_cpp_sip_LDFLAGS_OTHER ${qt_gui_cpp_LDFLAGS_OTHER} -Wl,-rpath,\\"lib\\")
 
+find_library(class_loader_LIBRARY class_loader)
+find_library(console_bridge_LIBRARY console_bridge)
+find_library(TinyXML2_LIBRARY tinyxml2)
```

Comment:
This is already done the the package level CMake file.

Reply:
It doesn't look like find_package(TinyXML2_vendor) is sufficient. I moved this to a find_package(TinyXML2) at the package level CMake file.

Reply:
Hmm, the variable set by find_package is TINYXML2_LIBRARY does not get a full path. find_library sets the full path to the library.